guides/veangle-guides/locking-angle.md

```diff
@@ -36,6 +36,10 @@ Once the lock has been created, to extend the lock expiration or add tokens, jus
 
 ![Increasing or adding tokens to lock](../../.gitbook/assets/claim-interest.png)
 
+### Expired lock
+
+Once a lock is expired, users need to execute a transaction to get their ANGLE token back from the lock. There will be a `Withdraw ANGLE` button on the [DAO App veANGLE](https://dao.angle.money/#/veANGLE) page, allowing you to get back your previously locked ANGLE tokens. 
```

Comment:
In this situation, there is a a `Withdraw ANGLE` button

Rather than There will be which gives the impression we don't have it yet